README.md

```diff
@@ -3,7 +3,7 @@
 `hauser` is a service to download FullStory data export files and load them into a data warehouse. (Redshift and BigQuery are the only warehouses supported currently. Others are easy to add -- pull requests welcome.)
 
 ## Quick Start
-
+**Note: Requires go 1.7 or higher**
```

Comment:
My personal preference is that this becomes a bullet point in the list below, rather than a note at the top.  So the first bullet point would be:

 * Make sure you have [installed](https://golang.org/doc/install) Go 1.7 or higher.

This is just personal preference, though.